data_model.md

```diff
@@ -226,10 +229,37 @@ for more detailed explanation of forecast parameters):
 Additional parameters, such as model configuration parameters, may be
 specified under the “extra parameters” key.
 
-
 Each Forecast has data associated with it. The format of this data is
 found in the Data Format section below
 
+#### Probabilistic forecasts
+{: .anchor}
+
+The Solar Forecast Arbiter supports the specification of a full
+probabilistic forecast in terms of a cumulative distribution function
+(CDF). The metadata for a probabilistic forecast container is the same
+as for a [single-valued forecast](#single-valued-forecasts) with the
+addition of two attributes:
+
+-   *Axis* - The axis on which the CDF is specified. *x* corresponds to
+    fixed variable values. *y* corresponds to fixed quantiles.
+-   *Points* - The fixed variable values or fixed quantiles at which
+    the forecasts will be made.
+
+The CDF may be specified in terms of its variable values (the
```

Comment:
Not sure about the restriction to fixed values, or fixed quantiles. Is this because the form of the CDF is defined up front, and then only a single vector of values (or quantiles) is uploaded at each time point?   I'd guess that most forecasters producing CDFs will stick with fixed quantiles at each forecast time, but perhaps not always.

An alternative would be to upload a sequence of (value, quantile) pairs at each time.  The length of the sequence could vary between time steps. The downside would be if we wanted to compare a specific quantile across time points.

In the end, the form of the CDF should support the metrics we will be calculating.  I suggest we revisit this specification once we have outlined the metrics.

Reply:
> Is this because the form of the CDF is defined up front, and then only a single vector of values (or quantiles) is uploaded at each time point? 

yes

> I'd guess that most forecasters producing CDFs will stick with fixed quantiles at each forecast time...

I agree.

> but perhaps not always.

Yes, this is why we wanted to provide the flexibility.

> An alternative would be to upload a sequence of (value, quantile) pairs at each time. The length of the sequence could vary between time steps.

We think this will be significantly harder to efficiently store, query, and analyze. While I don't know this for a fact, I would suspect that any probabilistic forecast users would also expect forecasters to  submit the same data every time. 

> In the end, the form of the CDF should support the metrics we will be calculating. I suggest we revisit this specification once we have outlined the metrics.

I agree, but I doubt that the metrics engagement will help much.